src/components/picker/examples/picker-empty-suggestions.tsx

```diff
@@ -1,12 +1,14 @@
 import { LimelPickerCustomEvent, ListItem } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
 
+// 👇 Do NOT copy this to production code! It's a MOCK for this example ONLY!
```

Comment:
I don't know, I'm a bit concerned that all these warning don't have a nice tone and assume the reader is stupid. We should be very clear and make it difficult to make this mistake (copying what should definitely not be copied), but in a friendly way if possible and without distracting too much from the example code.

Reply:
True…

I just didn't feel I had the time to do the better solution right away, but I agree. It's perhaps better to not merge this in the interim, and just do the better solution when someone has a little bit of time 👍